pkg/resources/network_policy.go

```diff
@@ -181,7 +182,7 @@ func UpdateNetworkPolicy(d *schema.ResourceData, meta interface{}) error {
 		for i, v := range newIps {
 			ipRequests[i] = *sdk.NewIPRequest(v)
 		}
-		setReq := sdk.NewNetworkPolicySetRequest().WithBlockedIpList(ipRequests)
```

Comment:
Please add a test in `network_policy_acceptance_test.go` to check the removal.

Reply:
Added.